app/assets/stylesheets/components/_list.sass

```diff
@@ -0,0 +1,8 @@
+$List-spacing: $Theme-spacing-small * 2
+$List-spacing-item: $Theme-spacing-small
+
+.List
+  padding: $List-spacing-item 0 $List-spacing 0
```

Comment:
Unless you're overriding any existing values on the left or right, there's no pointing in setting them here